package.json

```diff
@@ -12,10 +12,11 @@
   "typings": "build/src/index.d.ts",
   "scripts": {
     "build": "npm-run-all -p lint:tslint build:tsc -s build:webpack",
-    "build:tsc": "tsc -p test/",
+    "build:tsc": "tsc -p .",
```

Comment:
if you intend to compile them all at once, can you switch to only having one tsconfig project? that would make this more straightforward